pkg/build/builder/common.go

```diff
@@ -14,8 +14,8 @@ import (
 	"time"
 
 	"github.com/docker/distribution/reference"
-	dockercmd "github.com/docker/docker/builder/dockerfile/command"
-	"github.com/docker/docker/builder/dockerfile/parser"
+	dockercmd "github.com/openshift/imagebuilder/dockerfile/command"
+	"github.com/openshift/imagebuilder/dockerfile/parser"
```

Comment:
Why the change from docker/docker/builder to imagebuilder?

Reply:
imagebuilder incorporated it to avoid versioning conflicts, so the type we get from `imagebuilder.ParseDockerfile()` no longer matched the argument type for local functions like `nodeHasFromRef()`.